progenitor-impl/src/lib.rs

```diff
@@ -452,6 +452,14 @@ impl Generator {
                         let dur = std::time::Duration::from_secs(15);
 
                         reqwest::ClientBuilder::new()
+                            .default_headers(
```

Comment:
I think this won't cover the `new_with_client()` case, right?

Reply:
That does seem probably bad. My thought here--as I recall--was that it felt rude to modify a `reqwest::Client` that the user had provided to us, but I suppose that's both overly fussy **and** missing the most relevant cases. Thanks for taking a look.